tools/cli/src/index.ts

```diff
@@ -642,7 +670,7 @@ program
   .description('Shows resource types in a project')
   .argument(
     '<type>',
-    'resource types: attachments, card, cards, cardType, cardTypes, linkType, linkTypes, project, reports, template, templates, workflow, workflows',
+    'resource types: attachments, card, cards, cardType, cardTypes, labels, linkType, linkTypes, project, reports, template, templates, workflow, workflows',
```

Comment:
Is there any point to show details of one type? Probably not if it is just one string.

Reply:
Maybe if you wanted a list of all cards with a specific label.